docs/source/components/nodes/stereo_depth.rst

```diff
@@ -138,12 +162,7 @@ Currently configurable blocks
 Current limitations
 ###################
 
-If one or more of the additional depth modes (:code:`lrcheck`, :code:`extended`, :code:`subpixel`) are enabled, then:
-
-- median filtering is disabled on device
```

Comment:
afaik, this limitation has been resolved, right @szabi-luxonis ?

Reply:
yes, it's only disabled for subpixel bits 4 and 5.